src/components/templates/PosterHall/components/PosterPreview/PosterPreview.scss

```diff
@@ -66,30 +55,41 @@ $active-poster-shadows: 0 10px 20px 0 var(--primary--under-50a);
   &__categories {
     display: flex;
     flex-wrap: wrap;
+    margin-top: $spacing--md;
   }
 
   &__authorBox {
     display: flex;
     align-items: center;
-    margin-top: auto;
-  }
-
-  &__avatar {
-    margin-right: auto;
-  }
-
-  &__author {
-    margin-left: $spacing--sm;
     font-size: $font-size--sm;
-    font-weight: $font-weight--500;
-    @include line-clamp-with-overflow(5);
   }
 
-  &__contactEmail {
-    font-size: $font-size--sm;
+  &__thumbnailContainer {
+    flex-grow: 1;
+    // Tiny initial height forces the container to grow to fill remaining space
+    height: 1px;
+    background-size: cover;
+    background-position: center;
+    img {
+      max-width: 100%;
+      max-height: 100%;
+    }
   }
+}
 
-  &__moreInfoUrl {
-    font-size: $font-size--sm;
+.PosterAttendance {
+  margin: 0 0 0 auto;
+  background: opaque-black(0.3);
+  border-radius: $border-radius--xl;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-width: $online-container-min-width;
+  height: 30px;
+  padding: $spacing--md;
+  font-size: $font-size--sm;
+
+  &-icon {
```

Comment:
We should avoid use like this for ease of read/debug.

Cutoffs we use are `&__` and `&--` because it's harder to search in code for instances of `PosterAttendance-icon` and it also looks too similar to a typo bug i.e. code smell.

Reply:
Should this be `PosterAttendance-icon` or should it be `ScheduleItem__online-icon`?

Reply:
Resolved in 461139c50